### PR TITLE
ci: add gp7 support to merge ci

### PR DESCRIPTION
**Pull request by Sasasu**
CI part for #665

### CODE REVIEW

**Review by yihong0618 — APPROVED** [review]
LGTM

File: concourse/pipeline/res_def.yml [hunk 1]
```diff
@@ -274,6 +274,13 @@ resources:
     json_key: ((extension/extensions-gcs-service-account-key-dev2))
     versioned_file: intermediates_release/plcontainer/plcontainer-gppkg-ubuntu18-gp6.tar.gz
 
+- name: bin_plcontainer_gpdb7_rocky8_intermediates_rel
+  type: gcs
+  source:
+    bucket: gp-extensions-ci
+    json_key: ((extension/extensions-gcs-service-account-key-dev2))
+    versioned_file: intermediates_release/plcontainer/plcontainer-gppkg-rhel8-gp7.tar.gz
```
Comment (yihong0618) [inline concourse/pipeline/res_def.yml:282]: is there rocky8 binary here?

Comment (Sasasu) [reply concourse/pipeline/res_def.yml:282]: https://github.com/greenplum-db/plcontainer/blob/69fcf06104a67b6b856b99505d1a302d329cd145/concourse/scripts/entry.sh#L24-L44

rocky8 and rhel8 and all rhel8 compatible distributions will share the rhel8 name.

Comment (yihong0618) [reply concourse/pipeline/res_def.yml:282]: copy that

**Review by beeender — APPROVED** [review]

**Pull request merged**